modules/rosa-sts-byo-oidc.adoc

```diff
@@ -78,7 +78,7 @@ $ echo $OIDC_ID
 +
 [source,terminal]
 ----
-$ 30f5dqmk
+$ 13cdr6b
```

Comment:
These seem fine, though I wonder if you should have an example built out where it shows "export OIDC_ID=13cdr6b" and then you have this response? 

Same feedback about the "source,text" and `$` here.